package.json

```diff
@@ -2,7 +2,7 @@
   "author": "Dan Jenkins <dan.jenkins@holidayextras.com> (http://www.holidayextras.co.uk/)",
   "name": "geckoboard-push",
   "description": "Send data to Geckoboard via their new push API",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
could you take this back to 1.2 so it can go through the release process? but other than that this seems 100% right.